composer.json

```diff
@@ -16,16 +16,33 @@
         }
     ],
     "require": {
-        "php": ">=5.3.9",
+        "php": ">=5.5.9",
+        "symfony/console": "~3.3|~4.0",
+        "symfony/config": "~3.3|~4.0",
+        "symfony/dependency-injection": "~3.3|~4.0",
+        "symfony/framework-bundle": "~3.3|~4.0",
```

Comment:
require-dev i guess, although we extend from ContainerAwareCommand. Perhaps update to Command directly? Then its a require-dev for sure.